### PR TITLE
create stocks chart

### PR DESCRIPTION
**Pull request by Abdull-Rauf**

### CODE REVIEW

File: src/components/charts/StocksChart.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+type StocksChartScreenType = {
+  stocksData?: any;
+  priceData?: any;
+};
+
+const StocksChart: FC<StocksChartScreenType> = ({
+  stocksData,
+}): JSX.Element => {
+  const canvas = document.createElement("canvas");
```
Comment (Farhan-Nazir) [inline src/components/charts/StocksChart.tsx]: You don't need to create any canvas, chart library will create one for the component

File: src/services/stocks/getStocksData.service.ts [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-    }
+  period: string,
+  symbol: string
+): Promise<Record<string, any>> => {
+  const stocksEndpoint = getStocksEndpoint(period, symbol);
+  try {
+    const response = await axios.get(stocksEndpoint);
+    return response.data;
+  } catch (error) {
+    console.error();
```
Comment (Farhan-Nazir) [inline src/services/stocks/getStocksData.service.ts]: no console....

File: src/views/StocksChartScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    const data = abc.map((obj: any) => obj.priceData["2. high"]);
+    state.data = data.slice(0, 10);
+    state.labels = labels.slice(0, 10);
+    setState({
+      ...state,
+    });
+  }, [stocksData]);
+  return (
+    <div
+      style={{
```
Comment (Farhan-Nazir) [inline src/views/StocksChartScreen.tsx]: no in-line styling allowed, Please create scss module for styling

File: src/views/StocksChartScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import { FC, useState, useEffect } from "react";
```
Comment (Farhan-Nazir) [inline src/views/StocksChartScreen.tsx]: File name should be match your component name, if the folder name is `views` then file name should be `StockChart.view.tsx`. 
`StockChart` is not a correct name for page, A page can be `Home | Dashboard ` or something but the StockChart is component

File: src/components/charts/StocksChart.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        backgroundColor: gradient,
+        borderColor: "#31d0ad",
+        borderWidth: 0.5,
+        tension: 0.5,
+      },
+    ],
+  };
+
+  const options: any = {
+    title: {
```
Comment (Farhan-Nazir) [inline src/components/charts/StocksChart.tsx]: try this.
```suggestion
    options: {
        plugins: {
            title: {
                display: true,
                text: 'Custom Chart Title'
            }
        }
    }
```

File: src/components/charts/StocksChart.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+type StocksChartScreenType = {
+  stocksData?: any;
+  priceData?: any;
+};
+
+const StocksChart: FC<StocksChartScreenType> = ({
+  stocksData,
+}): JSX.Element => {
+  const canvas = document.createElement("canvas");
+  const ctx: any = canvas.getContext("2d");
```
Comment (Farhan-Nazir) [inline src/components/charts/StocksChart.tsx]: can you try some type here?

**Review by Farhan-Nazir — CHANGES_REQUESTED** [review]

File: src/components/charts/StocksChart.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  gradient.addColorStop(1, "#31d0ad");
+
+  const data = {
+    labels: stocksData.labels,
+    datasets: [
+      {
+        label: "High",
+        data: stocksData.data,
+        fill: true,
+        backgroundColor: gradient,
```
Comment (Farhan-Nazir) [inline src/components/charts/StocksChart.tsx]: can you try this?
```suggestion
        backgroundColor: 'linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(9,9,121,1) 35%, rgba(0,212,255,1) 100%)',
```

**Review by Farhan-Nazir — CHANGES_REQUESTED** [review]

File: src/components/charts/StocksChart.tsx (hunk 1 above)
Comment (Abdull-Rauf) [reply src/components/charts/StocksChart.tsx]: I tried but it didn't work for me.

File: src/components/charts/StocksChart.tsx (hunk 7 above)
Comment (Abdull-Rauf) [reply src/components/charts/StocksChart.tsx]: I have tried this but didn't work.

**Pull request merged**